### PR TITLE
mesa: depend on pygments

### PR DESCRIPTION
**Pull request by singingwolfboy**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
pre-existing linux failures

**Review by carlocab — DISMISSED** [review]

File: Formula/mesa.rb [hunk 1]
```diff
@@ -38,6 +38,7 @@ class Mesa < Formula
   depends_on "libxcb"
   depends_on "libxdamage"
   depends_on "libxext"
+  depends_on "pygments"
```
Comment (carlocab) [inline Formula/mesa.rb:41]: Should be build-only.

Comment (singingwolfboy) [reply Formula/mesa.rb]: Fixed!

**Comment by carlocab** [comment]
Cherry-picked onto #113528.

**Pull request closed without merging**